src/Backend/Modules/Slideshows/Actions/Edit.php

```diff
@@ -103,6 +105,12 @@ private function handleForm()
             'form-control danger title'
         );
 
+        $template = $this->frm->addDropdown(
```

Comment:
you should only show/add this dropdown if there are more than 1 possible template, just like we do in the Content Blocks module